openshift/topics/tutorials.adoc

```diff
@@ -675,8 +704,17 @@ See the link:https://access.redhat.com/documentation/en-us/red_hat_single_sign-o
 
 The client *Secret* is needed to configure OpenID-Connect on the OpenShift master in the next section. You can copy it now from under the *Credentials* tab. The secret is <pass:quotes[_7b0384a2-b832-16c5-9d73-2957842e89h7_]> for this example.
 
+[role="additional-resources"]
+.Additional resources
+* link:{project_doc_base_url}//server_administration_guide/index#assembly-managing-clients_server_administration_guide[Managing OpenID Connect and SAML Clients]
+
 ==== Configuring OpenShift Master for {project_name} Authentication
-Log in to the OpenShift master CLI. You must have the required permissions to edit the */etc/origin/master/master-config.yaml* file.
+Log in to the OpenShift master CLI. 
+
+.Prerequisites
+You must have the permissions to edit the */etc/origin/master/master-config.yaml* file.
+
+.Procedure
 
 . Edit the */etc/origin/master/master-config.yaml* file and find the *identityProviders*. For example, in the case the OpenShift master is configured with the link:{ocpdocs_htpasswd_idp_link}[HTPassword identity provider], the *identityProviders* section will look similar to the following one:
```

Comment:
Maybe replace `and find the *identityProviders*` with `and find the *identityProviders* section.`?